scripts/build/binary

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 
-source ./scripts/build/ldflags
+. ./scripts/build/ldflags
 
-go build -o ./build/docker --ldflags "${LDFLAGS}" github.com/docker/cli/cmd/docker
+CGO_ENABLED=0 go build -o ./build/docker --ldflags "${LDFLAGS}" github.com/docker/cli/cmd/docker
```

Comment:
Yes, but these scripts are invoked from `make` outside the Dockerfiles too so builds not using Docker won't work without this setting.